projects/cdk/utils/dom/test/is-element-editable.spec.ts

```diff
@@ -0,0 +1,35 @@
+import {tuiIsElementEditable} from '@taiga-ui/cdk';
+
+describe(`tuiIsElementEditable`, () => {
+    let element: HTMLElement;
+
+    beforeEach(() => {
+        element = document.createElement(`input`);
+    });
+
+    it(`should return true for editable textfield`, () => {
+        element.setAttribute(`type`, `text`);
+        expect(tuiIsElementEditable(element)).toBe(true);
+    });
+
+    it(`should return true for editable textarea`, () => {
+        element = document.createElement(`textarea`);
+        expect(tuiIsElementEditable(element)).toBe(true);
+    });
+
+    it(`should return false for readonly textfield`, () => {
```

Comment:
We should also check it returning false for `type="checkbox"` for example.

Reply:
why returning false?

Reply:
<img width="821" alt="image" src="https://github.com/Tinkoff/taiga-ui/assets/12021443/964c28eb-89fd-4c5f-8d0a-fa8c2530ce20">

but `tuiIsElementEditable` returning true

Reply:
done